config/initializers/inflections.rb

```diff
@@ -14,3 +14,6 @@
 # ActiveSupport::Inflector.inflections(:en) do |inflect|
 #   inflect.acronym 'RESTful'
 # end
+ActiveSupport::Inflector.inflections do |inflect|
```

Comment:
This was cool! Rails was automatically changing cafes to caves so this is how you change that which I hadn't had to use previously